### PR TITLE
Updated lists using new query to include vessels with >500 active poi…

### PR DESCRIPTION
**Pull request by davidkroodsma**
…nts in a year and reporting fishing >99% of the time

I recently explored what all the vessels who have very few positions are doing. Here are some of the data explorations:

 - [Points per Year for Fishing Vessels](https://github.com/GlobalFishingWatch/data-dev/blob/master/points_per_vessel/2016-11-10-points-per-year.md)
 - [Likely Fishing Cutoff](https://github.com/GlobalFishingWatch/data-dev/blob/master/points_per_vessel/2016-11-10-likely-fishing-cutoff.md)
 - [Fishing Activity of Low Activity Vessels]()

Our previous algorithm included likely fishing vessels as only those that have at least 1000 positions in a year. I found it was better to count "active positions" (positions with a speed > .1 knots), thus ignoring vessels that stayed at port, and lower the threshold to 500 positions in a year. This way we include roughly 95% of the total fishing effort. 

I also found that we should include vessels that broadcast that they are fishing vessels >99 percent of the time, as opposed to the previous 100% of the time. The reason is that due to errors in broadcast, some fishing vessels have a bad message where they don't broadcast that they are fishing vessels. Including these vessels increases the total number of fishing vessels by a few percentage points. 

This updated version increases the number of fishing vessels by ~10,000 per year. It is currently in the branch `version 1.0`, and needs to be merged to master.

### CODE REVIEW

File: treniformis/_assets/GFW/FISHING_MMSI/LIKELY/export.sh [hunk 1]
```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
```
Comment (bitsofbits) [inline treniformis/_assets/GFW/FISHING_MMSI/LIKELY/export.sh]: Is this file supposed to be here? I thought I banished all shell scripts to the outer darkness!

File: treniformis/_assets/GFW/FISHING_MMSI/LIKELY/test_query.sql [hunk 2]
```diff
@@ -0,0 +1,5 @@
+  SELECT
```
Comment (bitsofbits) [inline treniformis/_assets/GFW/FISHING_MMSI/LIKELY/test_query.sql]: Is this file supposed to be here?

Comment (davidkroodsma) [inline treniformis/_assets/GFW/FISHING_MMSI/LIKELY/test_query.sql]: @bitsofbits -- ah, yes, now I remember the other reason I didn't want to merge this yet. I should eliminate these.

**Comment by bitsofbits** [comment]
@davidkroodsma, it looks like you still need to pull my master, then merge it into your code: 
```
git checkout master
git pull
git checkout version1.0
git merge master
# fix conflicts
```
However, there is a decent chance this will cause a bunch a conflicts that you'll have to annoyingly fix by hand. So, it might be best to just create a new branch from the current master (after `git pull`) and copy your changes over.

**Comment by davidkroodsma** [comment]
@bitsofbits Yeah -- that sounds about right. Look for a new pull request soon from another branch.

**Pull request closed without merging**